.github/workflows/build-test-deploy.yml

```diff
@@ -136,6 +137,33 @@ jobs:
           command: nix -- build .#cargo-fmt-check
           token: "${{ secrets.CACHIX_AUTH_TOKEN }}"
 
+  check-nix-long:
+    name: "Check nix long"
+    runs-on: ubuntu-latest
+    needs:
+      - check-nix
+    concurrency:
+      group: ${{ github.workflow }}-check-nix-long-${{ github.ref }}
+      cancel-in-progress: true
+    container:
+      image: niteo/nixpkgs-nixos-22.05:316b762afdb9e142a803f29c49a88b4a47db80ee
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          echo "experimental-features = nix-command flakes" > /etc/nix/nix.conf
+          echo "narinfo-cache-negative-ttl = 0" >> /etc/nix/nix.conf
+      - uses: cachix/cachix-action@f5f67badd061acb62b5c6e25e763572ca8317004
+        with:
+          skipPush: true
+          installCommand: |
+            nix-channel --add ${{ env.NIX_NIXPKGS_CHANNEL }} nixpkgs
+            nix-channel --update
+            nix-env -iA nixpkgs.cachix
+          name: composable-community
+      - run: |
+          chmod +x ./.github/workflows/check-nix-long.sh
```

Comment:
this is extra
```
$ cat <<EOF > foo.sh
> export BAR=123
> EOF

$ echo $BAR
# empty string

$ source foo.sh 
$ echo $BAR
123
```

Reply:
you mean you propose to store sh in yaml where it is not listed neither testable without GH?

Reply:
or you propose to use `./` instead of `source ./`?

Reply:
> 

You can remove `chmod +x ./.github/workflows/check-nix-long.sh`. `source` doesn't require an executable flag.

Reply:
Assuming suggestion was to remove source. removed in next pr for zombient. thank you.